tests/conftest.py

```diff
@@ -57,6 +59,9 @@ def clean_all(route_network, vm_hostname=None):
     for hv in hvs:
         clean_hv(hv, pattern)
```

Comment:
if `datacenter_type` is `aws.dct` do we actually need to clean hvs then?

Reply:
No, we don't. Added `kvm.dct` here.